diktat-rules/src/main/kotlin/org/cqfn/diktat/ruleset/utils/AstNodeUtils.kt

```diff
@@ -62,8 +62,8 @@ fun ASTNode.hasChildOfType(elementType: IElementType): Boolean =
 fun ASTNode.hasAnyChildOfTypes(vararg elementType: IElementType): Boolean =
         elementType.any { this.hasChildOfType(it) }
 
-fun ASTNode.isBlockEmpty(node: ASTNode?) = node?.let { listOf(ElementType.LBRACE, ElementType.RBRACE, WHITE_SPACE)
-        .containsAll(node.getChildren(null).map { it.elementType })} ?: true
+fun ASTNode.isBlockEmpty(elementType: IElementType) = this.findChildByType(elementType)?.let { listOf(ElementType.LBRACE, ElementType.RBRACE, WHITE_SPACE)
+        .containsAll(this.findChildByType(elementType)!!.getChildren(null).map { it.elementType })} ?: true
```

Comment:
You don't need to pass element type here: just call this method on the node you need, like `node.findChildByType(BLOCK).isBlockEmpty`. You can change receiver of this extension to `ASTNode?`. Also, you should reuse list of node types. If this constant is needed here too, better declare it in utils.